contrib/diurnal/example-diurnal-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: diurnalcontroller
+  name: diurnalcontroller
+spec:
+  replicas: 1
+  selector:
+    name: diurnalcontroller
+  template:
+    metadata:
+      labels:
+        name: diurnalcontroller
+    spec:
+      containers:
+        - command: ["diurnal_math", "-ns", "default", "-labels", "name=redis-slave", "-sample", "300", "-countfn", "10+10*sin(t)"]
```

Comment:
The interface should not involve users typing "a+b*sin(c*t)".  There are way too many ways to make mistakes.  Instead, the diurnal controller should have parameters like:
 - min
 - peak
 - peak hour of day, in a standard time format.

Note that actual load curves are not sinusoidal.  They have steeper ramp up and ramp down, and flatter peaks and narrower troughs.

The other way to do it is piecewise linear with like a couple of (value, time-of-day) points for each day.